app/controllers/miq_policy_controller/policy_profiles.rb

```diff
@@ -43,7 +43,7 @@ def profile_edit
         policies.each { |p| current.push(p.id) } # Build an array of the current policy ids
         mems = @edit[:new][:policies].invert     # Get the ids from the member list box
         begin
-          policies.each { |c| profile.remove_member(MiqPolicy.find(c)) unless mems.include?(c.id) } # Remove any policies no longer in the members list box
+          policies.each { |c| profile.remove_member(c) unless mems.include?(c.id) } # Remove any policies no longer in the members list box
           mems.each_key { |m| profile.add_member(MiqPolicy.find(m)) unless current.include?(m) }    # Add any policies not in the set
```

Comment:
Is there a similar issue here with `MiqPolicy.find(m)`?

Reply:
@chessbyte this one is fine because `m` here is an id of a policy record